gabarit/template_num/num_project/tests/test_utils_models.py

```diff
@@ -590,7 +590,7 @@ def test10_predict(self):
         model, model_conf = utils_models.load_model(model_dir='test_model')
         # Check just the type, regression is complicated...
         self.assertEqual(type(utils_models.predict(x_test_2, model)), list)
-        self.assertTrue(isinstance(utils_models.predict(x_test_2_solo, model), (np.floating, float)))
+        self.assertTrue(isinstance(utils_models.predict(x_test_2_solo, model), list))
         remove_dir(model_dir)
 
 
```

Comment:
Add test with several inputs

Reply:
This is the test for mono input, tests with several inputs is above